package.json

```diff
@@ -64,11 +64,13 @@
     "@types/node": "^12.0.10",
     "@typescript-eslint/eslint-plugin": "^1.11.0",
     "@typescript-eslint/parser": "^1.11.0",
+    "chalk": "^2.4.2",
     "coveralls": "^3.0.4",
     "cross-env": "^5.2.0",
     "eslint": "^6.0.1",
     "eslint-plugin-import": "^2.18.0",
     "eslint-plugin-node": "^9.1.0",
+    "glob": "^7.1.4",
```

Comment:
glob is for?

Reply:
https://github.com/meriyah/meriyah/blob/7e8d0463a13d019d16fe4ec2906801e41edea36d/scripts/prettier.js#L13